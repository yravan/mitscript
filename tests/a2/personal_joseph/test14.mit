a = "this is a multiline
string test";

print(a);
