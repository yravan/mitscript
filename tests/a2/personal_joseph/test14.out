this is a multiline
string test
